google-maps/src/web.ts

```diff
@@ -178,6 +179,29 @@ export class CapacitorGoogleMapsWeb
     }
   }
 
+  async getMapBounds(_args: { id: string }): Promise<LatLngBounds> {
+    const bounds = this.maps[_args.id].map.getBounds();
+
+    if (!bounds) {
+      throw new Error('Google Map Bounds could not be found.');
+    }
+
+    return {
+      southwest: {
+        latitude: bounds.getSouthWest().lat(),
```

Comment:
When you switch to using LatLng, be sure to change it here as well (to use `lat` and `lng` instead of `latitude` and `longitude`.